projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.html

```diff
@@ -14,7 +14,7 @@
   </div>
   <ng-container *ngIf="selected">
     <textarea #textArea type="text" required name="content" [maxlength]="COMMENT_CHAR_LIMIT" (focusout)="onFocusOut()"
-              [ngClass]="commentStyle()" [(ngModel)]="fullComment" [readOnly]="!editable">
+              [ngClass]="commentStyle()" [(ngModel)]="fullComment" [readOnly]="!editable" aria-label="comment">
```

Comment:
should this also not have index like the annotations

Reply:
this ticket was only for a label - not ID like the other ticket